lib/bootloader_setup.pm

```diff
@@ -249,7 +250,24 @@ sub boot_local_disk {
         if (match_has_tag 'inst-slof') {
             diag 'specifying local disk for boot from slof';
             my $slof = 5;
-            $slof += 1 if get_var('VIRTIO_CONSOLE');
+            if (get_var('VIRTIO_CONSOLE') && is_ppc64le) {
```

Comment:
Don't need is_ppc64le since this is in the condition of 'if (get_var('OFW'))', OFW is for ppc64le.